oauth2-authorization-server/src/test/java/org/springframework/security/config/annotation/web/configurers/oauth2/server/authorization/OAuth2RefreshTokenGrantTests.java

```diff
@@ -127,6 +131,48 @@ public void requestWhenRefreshTokenRequestValidThenReturnAccessTokenResponse() t
 
 	}
 
+	@Test
+	public void requestWhenCustomProviderSettingsThenOk() throws Exception {
```

Comment:
Let's remove these tests since it will be covered in `OAuth2AuthorizationCodeGrantTests` (see previous comment)